LogicLayer/Notifications/Html/Raw/email-confirmation.html

```diff
@@ -0,0 +1,52 @@
+﻿<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta name="viewport" content="width=device-width"/>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <title>Actionable emails e.g. reset password</title>
+    <link href="styles.css" media="all" rel="stylesheet" type="text/css"/>
+</head>
+
+<body>
+
+<table class="body-wrap">
+    <tr>
+        <td></td>
+        <td class="container" width="600">
+            <div class="content">
+                <table class="main" width="100%" cellpadding="0" cellspacing="0">
+                    <tr>
+                        <td class="content-wrap">
+                            <table width="100%" cellpadding="0" cellspacing="0">
```

Comment:
Attributes `width`, `cellpadding` and `cellspacing` are obsolete in HTML5 and should not be used.

Reply:
Applies to all HTMLs in `Html/Raw`.

Reply:
Done